src/io.jl

```diff
@@ -165,15 +166,28 @@ function write(
     sch = Tables.schema(rows)
 
     # Determine geometry columns
-    isnothing(geom_columns) && error(
-        "Please set `geom_columns` kw or define `GI.geometrycolumns` for $(typeof(table))",
+    if !isnothing(geom_columns)  # backwards compatible
+        geometrycolumn = geom_columns
+    end
+    isnothing(geometrycolumn) && error(
+        "Please set the `geometrycolumn` kwarg or define `GI.geometrycolumns` for $(typeof(table))",
```

Comment:
maybe we can also mention setting the DataAPI metadata key?  If that makes sense I'll add a suggestion